flang/lib/Optimizer/HLFIR/Transforms/BufferizeHLFIR.cpp

```diff
@@ -12,6 +12,7 @@
 // on the heap.
 //===----------------------------------------------------------------------===//
 
+#include "flang/Lower/ConvertExpr.h"
```

Comment:
This feels like a circular dependency but I think it actually isn't (flang/lower depends upon FIRTransforms but not the HLFIR passes AFAICT).

@jeanPerier  do you think we should move `fir::getAdaptToByRefAttrName`?

Reply:
I wasn't a fan of this, but I couldn't immediately figure a better choice.

Reply:
Agreed this should be moved. flang/Optimizer/Dialect/FIROpsSupport.h is the place where special attribute are defined for FIR.